passbolt.rst

```diff
@@ -0,0 +1,45 @@
+.. _passbolt-section:
+
+========
+Passbolt
+========
+
+Passbolt_ is an open source credential platform for modern teams.
+
+.. _Passbolt: https://www.passbolt.com/
+
+Configuration
+=============
+
+Once installed, navigate to the application Settings page and fill in the
+required fields:
+
+- **Passbolt FQDN**: Enter the fully-qualified domain name where the Passbolt
+  web UI will be accessible. If the FQDN is already registered in the
+  public DNS, you can enable the **Let's Encrypt certificate** switch to
+  obtain a valid encryption certificate.
+
+- **Admin mail address**: Enter the mail address of the Passbolt Admin. It's
+  used for password recovery.
+
+.. warning::
+
+    If an incorrect FQDN is configured, there is no way to get the right Admin
+    URL anymore. The preferred way to recover from this situation is to
+    remove the incorrect Passbolt instance, then install and configure a new
+    one.
+
+After first configuration, the Admin registration link is shown. Please follow 
+the link to create the Admin account.
+
+Consider storing the Passbolt keys in a safe place to be able to restore your account if needed.
```

Comment:
Can we have an info box to show this as something important for the restauration or the change of FQDN, some explanations are needed

Reply:
Done.